sway-lib-std/src/message.sw

```diff
@@ -1,46 +1,44 @@
 library message;
 
-use ::alloc::alloc;
+use ::alloc::alloc_bytes;
+use ::bytes::Bytes;
 use ::outputs::{Output, output_count, output_type};
 use ::revert::revert;
-use ::vec::Vec;
 use ::error_signals::FAILED_SEND_MESSAGE_SIGNAL;
 
-/// Sends a message `msg_data` to `recipient` with a `coins` amount of the base asset
+/// Sends a message to `recipient` of length `msg_len` through `output` with amount of `coins`
 ///
 /// ### Arguments
 ///
 /// * `recipient` - The address of the message recipient
-/// * `msg_data` - Arbitrary length message data
-/// * `coins` - Amount of base asset to send
-pub fn send_message(recipient: b256, msg_data: Vec<u64>, coins: u64) {
+/// * `msg_data` - arbitrary length message data
+/// * `coins` - Amount of base asset sent
+pub fn send_message(recipient: b256, msg_data: Bytes, coins: u64) {
```

Comment:
Do you find the updated description better? I've actually changed it recently in my `__smo` PR but I see you're now changing it back to how it was before. My reasoning at the time was that the definition of this function has no `output` nor `msg_len`.

Reply:
I restored the improved comments, they are indeed better. :)